### ISSUE TITLE
Add .pytest_cache to .gitignore

### ISSUE THREAD

**Issue by RaiVaibhav** [coala/coala-bears#2326]
Add `.pytest_cache` to `.gitignore`

difficulty/newcomer

**Comment by sladyn98** [comment]
I would like to go ahead with this issue as a new comer
How do i proceed

**Comment by RaiVaibhav** [comment]
@sladyn98  please go to http://api.coala.io/en/latest/Developers/Newcomers_Guide.html 
It will guide you.

**Comment by MilenOrfeev** [comment]
Hello, could I be assigned to this issue?

**Comment by GlebBerj** [comment]
I would like to work on this issue

**Comment by sanshay** [comment]
please assign this issue

**Comment by sanshay** [comment]
i am not continuing this issue anymore please assign this to someone else

**Comment by utkarsh2102** [comment]
Since this task is not being taken up by someone else now, could I be assigned the same?

**Comment by jayvdb** [comment]
I've assigned it to @sanshay , as they already have a PR for it

**Comment by utkarsh2102** [comment]
@jayvdb, as per Sanshay's latest comment above, he isn't following up the issue anymore.
So could I be assigned the same now?

**Comment by jayvdb** [comment]
@sanshay , no, this issue is yours.  There are many like it, but this one is yours.

**Comment by PandaBoi** [comment]
Id like to be assigned this issue if it is still open
